### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by robertasstankevicius**
* Fix logging imports.
* Add logging of exception tracebacks to exception_middleware.
* Add logging about HTTP requests.
* Set request scheme for parent API Gateway urls depending on current protocol.

### CODE REVIEW

File: b_lambda_layer_common/source/python/util/http_call.py [hunk 1]
```diff
@@ -37,6 +37,15 @@ def call(method, url, fields=None, headers=None, **urlopen_kw) -> HTTPResponse:
         """
         http = urllib3.PoolManager()
 
+        logger.info(
```
Comment (ignaloidas) [inline b_lambda_layer_common/source/python/util/http_call.py:40]:
```suggestion
        logger.debug(
```
This information is not frequently needed and clutters log a lot.

File: b_lambda_layer_common/source/python/util/http_call.py [hunk 2]
```diff
@@ -37,6 +37,15 @@ def call(method, url, fields=None, headers=None, **urlopen_kw) -> HTTPResponse:
...
+            f'Calling endpoint: \n'
+            f'{method=}\n'
+            f'{url=}\n'
```
Comment (ignaloidas) [inline b_lambda_layer_common/source/python/util/http_call.py:43]: `method` and `url` should be on the first line to compact the information

**Review by ignaloidas — CHANGES_REQUESTED** [review]

File: b_lambda_layer_common/source/python/util/http_call.py (hunk 1 above)
Comment (laimonassutkus) [reply b_lambda_layer_common/source/python/util/http_call.py:40]: Agreed.

File: b_lambda_layer_common/source/python/util/http_call.py (hunk 2 above)
Comment (laimonassutkus) [reply b_lambda_layer_common/source/python/util/http_call.py:43]: Agreed.

Comment (laimonassutkus) [reply b_lambda_layer_common/source/python/util/http_call.py:43]: Btw. Missing dot at the end of the sentence.

File: b_lambda_layer_common/source/python/exceptions/exception_mapper.py [hunk 3]
```diff
@@ -54,6 +55,8 @@ class ExceptionMapper:
 
     @staticmethod
     def map_and_raise(exception: Union[str, Dict[str, Any], Exception]) -> None:
+        logger.info(f'Trying to map error: {exception}')
```
Comment (laimonassutkus) [inline b_lambda_layer_common/source/python/exceptions/exception_mapper.py:58]: Add a dot or preferably 3 dots.

**Review by laimonassutkus — CHANGES_REQUESTED** [review]
Ignas and Laimonas change requests.

File: b_lambda_layer_common/source/python/exceptions/exception_mapper.py (hunk 3 above)
Comment (ignaloidas) [reply b_lambda_layer_common/source/python/exceptions/exception_mapper.py:58]: No need for 3 dots IMO, doesn't change the message meaningfully

File: b_lambda_layer_common/source/python/util/http_call.py (hunk 2 above)
Comment (robertasstankevicius) [reply b_lambda_layer_common/source/python/util/http_call.py:43]: It's not a sentence anymore when everything is in new line. I don't see the need here for any dots.

**Review by laimonassutkus — APPROVED** [review]

**Pull request merged**